OpenRA.Mods.Common/Traits/Conditions/GrantConditionOnDeploy.cs

```diff
@@ -36,6 +36,9 @@ public class GrantConditionOnDeployInfo : PausableConditionalTraitInfo
 		[Desc("Can this actor deploy on slopes?")]
 		public readonly bool CanDeployOnRamps = false;
 
+		[Desc("Do we Deploy / Undeploy this actor as a group?")]
+		public readonly bool DeployAsGroup = false;
```

Comment:
I don't like the notation tbh and the description doesn't help either. Can't think of anything better atm, but I had to look at the gif to realize what is this about.

You can be lengthy with "DeploySelectionToConsistentState" or something along that line (or mention this in the description).

Reply:
What do you think about the property name for DeployAsGroup? Since that's what's going to go in the YAML?

Reply:
I changed this to Synchronize
`
		[Desc("Does this actor need to synchronize it's deployment with other actors?")]
		public readonly bool SynchronizeDeployment = false;`

Reply:
Fine by me.